main.md

```diff
@@ -821,11 +821,11 @@ because the confirmation of the DPoP binding in the access token failed:
 !---
 ```
  HTTP/1.1 401 Unauthorized
- WWW-Authenticate: DPoP error="invalid_token",
+ WWW-Authenticate: DPoP error="invalid_dpop_proof",
    error_description="Invalid DPoP key binding", algs="ES256"
```

Comment:
This example was meant to show an error to for a token that was rejected because the confirmation of the DPoP binding in the access token failed - i.e. the proof validated fine but didn't match up with the cnf/jkt value.  And that would be invalid_token. Changing this to invalid_dpop_proof would also need the error_description to change as well as the text proceeding this example.

Reply:
ah then I misunderstood

Reply:
is there something that could be added/changed to avoid misunderstanding?

Reply:
Maybe the figure description, i mean - the DPoP Proof on its own was validated. The Access Token is not invalid, its the valid DPoP Proof that failed to confirm possession of the cnf claim. So what is it that is invalid? The access token, the proof, or both together?

Reply:
The way I think about it, it's the access token that's invalid. Validation of one of it's claims failed, which happened to be the cnf claim, but is the same outcome as if validation of exp, aud, etc. failed. 

I'll have to think about making this more clear. Or changing the premise of the example.